examples/examples_test.go

```diff
@@ -167,6 +167,8 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"frontend-service":        &api.Service{},
 			"redis-master-service":    &api.Service{},
 			"redis-slave-service":     &api.Service{},
+			"redis-slave":             nil,
```

Comment:
If don't add this, the examples_test.go will fail. Now the examples_test.go don't support test List Kind config file. This is a temporary change. We should make examples_test.go can test List Kind or `---` separate file. What do you think we should use List or `---` separate? And another thing is, although we plan to use a combine file instead of separate files like redis-slave-service, redis-slave-controller, I also keep them. Because some tests will use.